content/v1.0.0-beta.2/index.md

```diff
@@ -132,7 +132,7 @@ It discourages moving fast in a disorganized way. It helps you be able to move f
 
 ### Might Conventional Commits lead developers to limit the type of commits they make because they'll be thinking in the types provided?
 
-Conventional Commits encourages us to make more of certain types of commits such as fixes. Other than that, the flexibility of Conventional Commits allows your team to come up with their own types and change those types over time.
+Conventional Commits encourages us to make more of certain types of commits such as `fixes`. Other than that, the flexibility of Conventional Commits allows your team to come up with their own types and change those types over time.
```

Comment:
Should this be `fix`?

Reply:
I guess it's still `fixes` because the context said **more types** and `fix` is already shown up above

Reply:
mmm dunno about that one, you may open an issue and let's discuss it all together.
If you remove this one we can merge the dead links fix 👍

Reply:
Yeah, I think the original sentence is wrong. Maybe it should say

Conventional Commits encourages us to make more of certain types of commits such as `improvement` or `cleanup`.

Reply:
reverted